### PR TITLE
fix: airnode-protocol README and scripts updates

### PR DESCRIPTION
**Pull request by dcroote**
With relatively fresh eyes I identified updates needed within the `airnode-protocol` README and scripts.

### CODE REVIEW

File: packages/airnode-protocol/README.md [hunk 1]
```diff
@@ -21,8 +21,13 @@ yarn run test:coverage
 yarn run test:gas
 ```
 
-Deploy the contracts on a network (`<NETWORK>` must be one of the names from `credentials.example.json`), verify the
-deployment
+### Contract deployment
+
+To deploy the contracts on a network first copy `credentials.example.json` to `.env`. Enter a mnemonic and any
+appropriate API keys. Note that the mnemonic must be present (though can be arbitrary) even if the exclusive goal is
+contract verification. The `<NETWORK>` in the below commands must correspond to a network in `TODO`. If a hardhat error
```
Comment (dcroote) [inline packages/airnode-protocol/README.md:28]: What is our source of truth for network names?

File: packages/airnode-protocol/README.md [hunk 2]
```diff
@@ -21,8 +21,13 @@ yarn run test:coverage
 yarn run test:gas
 ```
 
-Deploy the contracts on a network (`<NETWORK>` must be one of the names from `credentials.example.json`), verify the
-deployment
+### Contract deployment
+
+To deploy the contracts on a network first copy `example.env` to `.env`. Enter a mnemonic and any appropriate API keys.
+Note that the mnemonic must be present (though can be arbitrary) even if the exclusive goal is contract verification.
+The `<NETWORK>` in the below commands must correspond to a network in `TODO`. If a hardhat error occurs during
```
Comment (dcroote) [inline packages/airnode-protocol/README.md:28]: What is our source of truth for network names? The file that was being referred to (`credentials.example.json`) does not exist...

Comment (bbenligiray) [reply packages/airnode-protocol/README.md:28]: Aliases from https://github.com/api3dao/chains

File: packages/airnode-protocol/scripts/generate-references.ts [hunk 3]
```diff
@@ -30,4 +30,4 @@ const networks = Object.entries(references.chainNames).reduce((acc, [chainId, na
 }, {});
 references.networks = networks;
 
-fs.writeFileSync(path.join('deployments', 'references.json'), JSON.stringify(references, null, 2));
+fs.writeFileSync(path.join('deployments', 'references.json'), JSON.stringify(references, null, 2) + '\n');
```
Comment (bbenligiray) [inline packages/airnode-protocol/scripts/generate-references.ts:33]: :brain:

**Review by bbenligiray — APPROVED** [review]

File: packages/airnode-protocol/README.md (hunk 2 above)
Comment (bbenligiray) [reply packages/airnode-protocol/README.md:28]: To make it easier, file names from https://github.com/api3dao/chains/tree/main/chains

**Pull request merged**